pom.xml

```diff
@@ -251,6 +251,13 @@
                 <scope>test</scope>
                 <version>${jsonassert.version}</version>
             </dependency>
+
+            <!-- KF-->
+            <dependency>
+                <groupId>org.apache.commons</groupId>
+                <artifactId>commons-math3</artifactId>
+                <version>${apache.commons.math3.version}</version>
```

Comment:
There is already a commons.math3 version that is being used. Please remove duplicate entries and use one version of the library for all extensions within the siddhi repo.